common/include/pcl/point_cloud.h

```diff
@@ -315,9 +315,12 @@ namespace pcl
       /** \brief Return an Eigen MatrixXf (assumes float values) mapped to the specified dimensions of the PointCloud.
         * \note This method is for advanced users only! Use with care!
         *
-        * \attention Since 1.4.0, Eigen matrices are forced to Row Major to increase the efficiency of the algorithms in PCL
-        *   This means that the behavior of getMatrixXfMap changed, and is now correctly mapping 1-1 with a PointCloud structure,
-        *   that is: number of points in a cloud = rows in a matrix, number of point dimensions = columns in a matrix
+        * \attention Comile time flags used for Eigen might affect the dimension of the Eigen::Map returned. If Eigen
```

Comment:
Comile -> Compile (same below)
I am not sure that there is a compile time flag that changes the default storage order, though? I think `Eigen::MatrixXf::Flags` is always set to col-major, unless the Eigen maintainers decide to change that for some reason?

Reply:
Eigen is a head only library. So the flag needs to be present at compile time.

Reply:
I thought there was no flag/option to change the default storage order, but turns out there is: `EIGEN_DEFAULT_TO_ROW_MAJOR` ([described here](https://eigen.tuxfamily.org/dox/TopicPreprocessorDirectives.html))